### PR TITLE
add rorcual validator

### PR DESCRIPTION
**Pull request by Rorcual-Nodes**

### CODE REVIEW

File: stewards.json [hunk 1]
```diff
@@ -127,4 +127,8 @@
         "ca": "Golden_Ratio_Staking/server_ca.crt",
         "endpoint": "https://sommelier.goldenratiostaking.net:5734"
     }
```
Comment (cbrit) [inline stewards.json:129]: Need a comma after the golden ratio staking entry, please!

**Review by cbrit — CHANGES_REQUESTED** [review]
Thanks for the PR. One small change needed!

**Comment by Rorcual-Nodes** [comment]
Done! Sorry for the typo.

**Review by cbrit — APPROVED** [review]
LGTM

**Pull request merged**